src/components/CommentCard.jsx

```diff
@@ -1,7 +1,30 @@
-const CommentCard = ({comment})=> {
+import { useEffect, useState } from "react"
+import { deleteComment } from "../utils/api_funcs"
+
+const CommentCard = ({comment, user})=> {
+    const [removeComment, setRemoveComment] = useState(false)
+    const [deleteComplete, setDeleteComplete] = useState(false)
+
+    const handleClick = () => {
+        setRemoveComment(true)
```

Comment:
It would make more sense for the code inside the useEffect, to be inside here. You're essentially just making a function, to trigger another, when the code could just be in the first function.